react/components/Profile/ProfileFormBox.tsx

```diff
@@ -9,6 +9,7 @@ import { Button } from 'vtex.styleguide'
 import ContentBox from '../shared/ContentBox'
 import UpdateProfile from '../../graphql/updateProfile.gql'
 import { withSettings, Settings } from '../shared/withSettings'
+import className from "../../styles/ContentBox.css";
```

Comment:
```suggestion
import className from '../../styles/ContentBox.css'
```

This is probably a formatting problem with your prettier: 

https://www.jetbrains.com/help/idea/prettier.html

https://www.jetbrains.com/help/idea/eslint.html